app/views/user_mailer/reset_password_instructions.html.erb

```diff
@@ -1,3 +1,27 @@
+<% if @pending_profile_requires_verification %>
+  <table>
+    <tbody>
+    <tr>
+      <td width="140">
+        <%= image_tag('letter-warning.svg', width: 140, height: 140, alt: '') %>
+      </td>
+      <td class="padding-left-3">
+        <h2 style="font-size:22px;"><%= t('user_mailer.reset_password_instructions.gpo_letter_header') %></h2>
+        <p><%= t('user_mailer.reset_password_instructions.gpo_letter_description') %></p>
+      </td>
+    </tr>
+    <tr>
+      <td class="s30" style="height: 30px"></td>
+    </tr>
+    <tr>
+      <td class="border-top border-primary-light s30" style="margin-top: 3em; margin-bottom: 3em; height: 30px;" colspan="2" align="center"></td>
+    </tr>
+    </tbody>
+  </table>
+  <% end %>
+  <h1>
+    <%= @header || message.subject %>
+  </h1>
```

Comment:
If we only `@hide_title` if it's `@pending_profile_requires_verification`, should we also only output the repositioned heading if it's `@pending_profile_requires_verification`?

Otherwise, I expect there'd be two headings in the scenario that the email is being sent for non-pending profile?

Reply:
Alternatively, we could always hide the default title (`@hide_title = true`) and then leave this as-is.

Reply:
You are correct -- that now belongs in the conditional block